packages/quicktype-core/src/language/Golang/GolangRenderer.ts

```
@@ -90,6 +90,42 @@ export class GoRenderer extends ConvenienceRenderer {
     }
+    
+    private lowercaseFirstLetter(str: string): string {
+        if (!str) return str;
+        return str.charAt(0).toLowerCase() + str.slice(1);
+    }
+
+    private emitConstructor(name: Name, c: ClassType): void {
+        this.ensureBlankLine();
+        let structInitialisationRows: Sourcelike[][] = [];
+        let ctorFunctionParameters: Sourcelike[][] = [];
+        this.forEachClassProperty(c, "none", (name, _, p) => {
```

ESLint

[error] 'name' is already declared in the upper scope on line 99 column 29.

(@typescript-eslint/no-shadow)